test/settings.test.ts

```diff
@@ -37,67 +42,97 @@ describe('StakingPool.settings', () => {
   beforeEach(async () => {
     testEnv = await loadFixture(fixture);
     await testEnv.stakingAsset.connect(depositor).faucet();
+    await testEnv.rewardAsset.connect(deployer).faucet();
+    await testEnv.rewardAsset.connect(deployer).approve(testEnv.stakingPool.address, RAY);
   });
 
   it('staking pool deployed', async () => {
     expect(await testEnv.stakingPool.stakingAsset()).to.be.equal(testEnv.stakingAsset.address);
-    expect(await testEnv.stakingPool.rewardAsset()).to.be.equal(testEnv.rewardAsset.address);
   });
 
   context('when the staking pool deployed', async () => {
-    it('reverts if general account initiate the round', async () => {
+    it('reverts if general account initiate the pool', async () => {
+      await testEnv.rewardAsset.connect(depositor).faucet();
+      await testEnv.rewardAsset.connect(depositor).approve(testEnv.stakingPool.address, RAY);
       await expect(
         testEnv.stakingPool
           .connect(depositor)
-          .initNewRound(rewardPerSecond, startTimestamp, duration)
+          .initNewPool(rewardPerSecond, startTimestamp_1, duration, testEnv.rewardAsset.address)
       ).to.be.revertedWith('OnlyAdmin');
     });
 
     it('success', async () => {
       await testEnv.stakingPool
         .connect(deployer)
-        .initNewRound(rewardPerSecond, startTimestamp, duration);
+        .initNewPool(rewardPerSecond, startTimestamp_1, duration, testEnv.rewardAsset.address);
 
       const poolData = await testEnv.stakingPool.getPoolData(1);
 
       expect(poolData.rewardPerSecond).to.be.equal(rewardPerSecond);
       expect(poolData.rewardIndex).to.be.equal(WAD);
-      expect(poolData.startTimestamp).to.be.equal(startTimestamp);
-      expect(poolData.endTimestamp).to.be.equal(endTimestamp);
+      expect(poolData.startTimestamp).to.be.equal(startTimestamp_1);
+      expect(poolData.endTimestamp).to.be.equal(endTimestamp_1);
+      expect(poolData.totalPrincipal).to.be.equal(0);
+      expect(poolData.lastUpdateTimestamp).to.be.equal(startTimestamp_1);
+      expect(await testEnv.stakingPool.currentPoolID()).to.be.equal(1);
+    });
+    it('success when pool is opened at the same time', async () => {
```

Comment:
What is the purpose of this test case and below? Because I can't find the logic that has a dependency on the timestamp in `initNewPool`.